### PR TITLE
Google map implementation

### PR DESCRIPTION
**Pull request by CSumm**
Resubmission of Google Maps implementation that includes:
-Google Map as minimap React component that uses google-map-react
-MiniMap stories.js file that can be used with storybook

Question along with this PR is: are component tests mandatory? I can try to add one, but I have not done it before and not sure which library we are using for it.

### CODE REVIEW

File: src/app/component/Minimap/MiniMap.stories.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import MiniMap from './MiniMap';
+
+storiesOf("Components | Button", module)
```
Comment (mat10tng) [inline src/app/component/Minimap/MiniMap.stories.js]: We are now trying to write stories the same as Input Component, with Knobs decorator

File: src/app/component/Minimap/MiniMap.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+// Module imports
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import GoogleMapReact from "google-map-react";
+import styled from 'styled-components';
+/*** [end of imports] ***/
+
+
+//size variations for maps
+const SmallMap = styled.div`
```
Comment (mat10tng) [inline src/app/component/Minimap/MiniMap.jsx]: style  should go into MiniMap.style.js

**Comment by mat10tng** [comment]
Hi! there is some conflicts of package.json, once you solve that, I will merge in the pr

File: src/app/component/Minimap/MiniMap.stories.js (hunk 1 above)
Comment (CSumm) [reply src/app/component/Minimap/MiniMap.stories.js]: I added knobs decorator, however I noticed when changing inside of storybook, the props do not resize right away. It only resize if you write in stories.js code

File: src/app/component/Minimap/MiniMap.jsx (hunk 2 above)
Comment (CSumm) [reply src/app/component/Minimap/MiniMap.jsx]: Moved to new file called MiniMap.style.js

**Comment by CSumm** [comment]
I feel I might be missing something from the test however I am not entirely sure. A review would help. I'll also try and study Jest a bit more.

**Comment by mat10tng** [comment]
we are using other maps now?

**Comment by CSumm** [comment]
We'll use other map, but will still use Google's geocoding api to convert addresses to latitude and longitude. Since its a later priority, I can hold off with this PR for now

**Pull request closed without merging**